tests/caasp/stack_kubernetes.pm

```diff
@@ -20,7 +20,9 @@ use testapi;
 sub run {
     # Use downloaded kubeconfig to display basic information
     switch_to 'xterm';
+    assert_script_run 'mv ~/Downloads/kubeconfig ~/.kube/config';
```

Comment:
This will fail - file is moved in download_kubeconfig, so ~/Downloads/kubeconfig does not exist now